modules/olm-updating-index-image.adoc

```diff
@@ -3,12 +3,30 @@
 // * operators/admin/olm-managing-custom-catalogs.adoc
 // * operators/admin/olm-restricted-network.adoc
 
+ifdef::openshift-origin[]
+:index-image-pullspec: quay.io/operator-framework/upstream-community-operators:latest
+:index-image: upstream-community-operators
```

Comment:
I only see the `{index-image}` being used in this module. Are the other attributes coming at some point in the future?

Reply:
Unlikely, was debating whether to bring them all along together or just use the ones I need. Think I'll trim it down.